README.md

```diff
@@ -113,30 +119,38 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
 
 ## Latest release
 
-🎉 KubeSphere 3.2.0 is now available! See the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
-
+🎉 KubeSphere 3.2.0 is now available! It has supported up to Kubernetes 1.22, see the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
 ## Installation
 
 KubeSphere can run anywhere from on-premise datacenter to any cloud to edge. In addition, it can be deployed on any version-compatible Kubernetes cluster.
-
 ### Quick start
+#### Installing on K8s/K3s
 
-1. Run the following commands to install KubeSphere on an exiting Kubernetes cluster:
+Run the following commands to install KubeSphere on an exiting Kubernetes cluster:
 
 ```yaml
 kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.2.0/kubesphere-installer.yaml
    
 kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/v3.2.0/cluster-configuration.yaml
 ```
+#### All-in-one
 
-2. You can run the following command to view the installation logs. After KubeSphere is successfully installed, you can use `http://IP:30880` to access the KubeSphere Console with the default account and password (admin/P@88w0rd).
+👨‍💻 No Kubernetes? You can install a single-node Kubernetes/K3s and KubeSphere on your Linux machine with [KubeKey](https://github.com/kubesphere/kubekey). Let's take K3s as an example:
 
 ```yaml
-kubectl logs -n kubesphere-system $(kubectl get pod -n kubesphere-system -l app=ks-install -o jsonpath='{.items[0].metadata.name}') -f
+# Download KubeKey
+curl -sfL https://get-kk.kubesphere.io | VERSION=v1.2.0 sh -
+# Make kk executable
+chmod +x kk
+# Create a cluster
+./kk create cluster --with-kubernetes v1.21.4-k3s --with-kubesphere v3.2.0
 ```
 
-> 👨‍💻 No Kubernetes cluster? Try [All-in-one](https://kubesphere.io/docs/quick-start/all-in-one-on-linux/) to install a single-node Kubernetes and KubeSphere on your Linux machine. 
+You can run the following command to view the installation logs. After KubeSphere is successfully installed, you can use `http://IP:30880` to access the KubeSphere Console with the default account and password (admin/P@88w0rd).
```

Comment:
you can access the KubeSphere web console at `http://IP:30880` and log in using the default administrator account (admin/P@88word).

Reply:
Done